docs/eventing/samples/apache-rocketmq-source/sample/apacherocketmqsource.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: sources.eventing.knative.dev/v1alpha1
```

Comment:
this is the one you want here

Reply:
This file has been deleted and can be found here https://github.com/apache/rocketmq-externals/blob/master/rocketmq-knative/source/sample/apacherocketmqsource.yaml